snippets/cpp.snippets

```diff
@@ -138,6 +138,11 @@ snippet ns
 	namespace ${1:`vim_snippets#Filename('', 'my')`} {
 		${0}
 	} /* namespace $1 */
+# anonymous namespace
+snippet ans
+	namespace {
+		${0}
+	} /* anonymous namespace */
```

Comment:
Is this commented needed?

Reply:
Which comment are you referring to? `# anonymous namespace` is not needed. But `/* anonymous namespace */` should be there at least for consistency with the snippet 'ns'.

Reply:
Could we remove it from both please?